dgen_os/python/input_data_functions.py

```diff
@@ -373,31 +408,44 @@ def function(df):
 
 
 #%%
-def import_agent_file(scenario_settings, con, cur, engine, model_settings, agent_file_status, input_name):
+def import_agent_file(scenario_settings, con, model_settings, agent_file_status, input_name):
     """
     Generates new agents or uses pre-generated agents from provided .pkl file
```

Comment:
```suggestion
    Imports pre-generated agents from provided .pkl file
```
We do not support generating the agent file so we can remove mention of that option. Can also update the overall function to remove the if/else statement